docs/control-structures.rst

```diff
@@ -273,18 +273,20 @@ i.e. the following is not valid: ``(x, uint y) = (1, 2);``
 Complications for Arrays and Structs
 ------------------------------------
 
-The semantics of assignments are a bit more complicated for
-non-value types like arrays and structs.
-Assigning *to* a state variable always creates an independent
-copy. On the other hand, assigning to a local variable creates
-an independent copy only for elementary types, i.e. static
-types that fit into 32 bytes. If structs or arrays (including
-``bytes`` and ``string``) are assigned from a state variable
-to a local variable, the local variable holds a reference to
-the original state variable. A second assignment to the local
-variable does not modify the state but only changes the
-reference. Assignments to members (or elements) of the local
-variable *do* change the state.
+The semantics of assignments are more complicated for non-value types like arrays and structs,
+including ``bytes`` and ``string``.
+
+Assigning **to a state variable** always creates an independent copy.
+
+Assigning **to a local variable** with the same data location creates an independent copy only for
+elementary types, i.e. static types that fit into 32 bytes.
```

Comment:
Since this is under the heading "Complications for arrays and structs", anything that does not affect arrays and structs e.g., sentences about elementary (or value types) may be moved to a different section.

Reply:
I agree with you here I think, but when I thought, "i should remove or reword this", I was stuck with thinking, "replace with what?"